apps/server/src/services/report-service/ReportService.ts

```diff
@@ -21,13 +21,25 @@ export function clear(id?: string) {
 }
 
 export function eventStart(state: RuntimeState) {
+  // eslint-disable-next-line no-unused-labels -- dev code path
+  DEV: {
+    if (state.eventNow === null) {
+      throw new Error('report.eventStart: called without eventNow present');
+    }
+  }
+
   report.set(state.eventNow.id, { ...blankReportData, startAt: state.timer.startedAt });
 }
 
 export function eventStop(state: RuntimeState) {
-  if (state.eventNow?.id === undefined) {
-    return;
+  // eslint-disable-next-line no-unused-labels -- dev code path
+  DEV: {
+    if (state.eventNow === null) {
+      throw new Error('report.eventStop: called without eventNow present');
+    }
   }
+
+  //TODO: How should pauses be calculated into the report?
```

Comment:
I think the reporter doesnt need to know the details of what happened inside a timer.
It would be sufficient to have startedAt and finishedAt timestamps